site/src/Librecores/ProjectRepoBundle/Doctrine/ProjectMetricsProvider.php

```diff
@@ -238,4 +267,290 @@ function (LanguageStat $a, LanguageStat $b) {
 
         return $result;
     }
+
+    /**
+     * Get a code quality score calculated from various other metrics
```

Comment:
Please add the algorithm used in the phpdoc so that people don't need to reconstruct the formula based on the code. Or add a link to other documentation where this formula is documented.